src/Settings/About.as

```diff
@@ -65,6 +65,7 @@ void RenderAboutTab()
     if (UI::RedButton(Icons::Heart + " \\$zSupport ManiaExchange")) OpenBrowserURL("https://"+MX_URL+"/support");
 
     UI::Text("Base URL \\$777" + MX_URL);
+    if (PluginSettings::RMC_MX_Url != "https://"+MX_URL) UI::Text("Proxy URL \\$777" + PluginSettings::RMC_MX_Url);
```

Comment:
Change "Proxy URL" by "Base URL" (the same way as Openplanet's Plugin Manager did)
![image](https://github.com/GreepTheSheep/openplanet-MXRandom/assets/42576124/284c8e97-b8c4-4719-93e5-ce7e8f082a78)

Reply:
removed the second section altogether.